lib/load-config.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const path = require('path');
+const fs = require('fs');
+// ESM seems to break `path` and `fs`, require them first before setting it up..
+const requireESM = require('esm')(module);
+const pkgConf = require('pkg-conf');
+
+const loadConfigFile = projectDir => {
+	const loaded = requireESM(path.resolve(projectDir, 'ava.config.js'));
+	const config = (loaded.__esModule) ? loaded.default : loaded;
+	if (config.then) {
+		throw new Error('Config file must not return a promise');
+	}
+	return (typeof config === 'function') ? config(projectDir) : config;
```

Comment:
Let's pass `{projectDir}`, so we can extend this with other options later.

Reply:
✅

Reply:
Ah, I meant doing `config({projectDir})`, not `loadConfigFile({projectDir})`.